### PR TITLE
add lecture from time & lecture until

### LINKED ISSUE #81: add lecture time and duration

### PR DESCRIPTION
**Pull request by DavidBadura**
fix #81

### CODE REVIEW

File: src/js/components/DateDiff.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import moment from "moment";
+
+export default class DateDiff extends React.Component {
+    render() {
+        let from = moment(this.props.from);
+        let until = moment(this.props.until);
+
+        let diff = until.diff(from, 'minutes');
+
+        console.log(diff);
```
Comment (tolry) [inline src/js/components/DateDiff.jsx]: should be removed?

**Comment by tolry** [comment]
:+1: 

note: a quick screenshot of the UI would have been nice (no need for me to pull and install the changes to judge them :smile: )

**Comment by DavidBadura** [comment]
cc @tolry 

bildschirmfoto von 2016-08-22 12-11-26

bildschirmfoto von 2016-08-22 12-11-38

**Comment by tolry** [comment]
@DavidBadura great, thx

**Pull request merged**